src/controllers/ArticleDislikeController.js

```diff
@@ -0,0 +1,61 @@
+import status from '../config/status';
```

Comment:
consider using toggle approach in liking and disliking, instead of checking if the article is already liked or disliked.